src/libclipper/src/redis.cpp

```diff
@@ -438,6 +449,22 @@ bool add_application(redox::Redox& redis, const std::string& appname,
   }
 }
 
+bool add_model_links(redox::Redox& redis, const std::string& appname,
+                     const std::vector<std::string>& model_names) {
+  if (send_cmd_no_reply<string>(
+          redis, {"SELECT", std::to_string(REDIS_APP_MODEL_LINKS_DB_NUM)})) {
+    for (auto model_name : model_names) {
```

Comment:
@dcrankshaw Any advice here?

It's true that this is fine for now because of the restriction that an application may only have one backing model. However, it would be nice to write this efficiently for future use cases when the restriction is loosened.